ArtOfIllusion/src/artofillusion/keystroke/KeystrokeManager.java

```diff
@@ -218,13 +210,14 @@ public static void saveRecords() throws Exception
 
     File dir = ApplicationPreferences.getPreferencesDirectory();
     File outFile = new File(dir, KEYSTROKE_FILENAME);
-    OutputStream out = new BufferedOutputStream(new SafeFileOutputStream(outFile, SafeFileOutputStream.OVERWRITE));
-    DOMSource source = new DOMSource(doc);
-    StreamResult result = new StreamResult(out);
-    TransformerFactory transFactory = TransformerFactory.newInstance();
-    Transformer transformer = transFactory.newTransformer();
-    transformer.setOutputProperty(OutputKeys.INDENT, "yes");
-    transformer.transform(source, result);
-    out.close();
+    try (OutputStream out = new BufferedOutputStream(new SafeFileOutputStream(outFile, SafeFileOutputStream.OVERWRITE)))
```

Comment:
Why the new `try` block? It's been functional for years with no trouble that I'm aware of...